src/app/core/service/account/account.service.ts

```diff
@@ -0,0 +1,19 @@
+import {Inject, Injectable} from '@angular/core';
+import {AuthConfig, AuthTokenConfig} from '@core';
+import {HttpClient} from '@angular/common/http';
+import {Observable} from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AccountService {
+
+  constructor(@Inject(AuthConfig) private config: AuthTokenConfig,
```

Comment:
`AuthConfig` should only be included in `auth.service.ts`.
Please, consider creating a `getConfig` or a getter inside the AuthService and consume that method to read the configuration object.
Eg.
```
account.service.ts

import { AuthService } from '../auth.service';
...
export class AccountService {
	constructor(private authService: AuthService) {}

	public getUserAccounts(): Observable<Array<Account>> {
		return this.http
      		.get<Array<Account>>(`${this.authService.getConfig().ApiUrl}/protected/v1/accounts`);
	}
}
```

```
auth.service.ts

...

export class AuthService {
	public getConfig() {
		return this.config;
	}
}
```